src/action/common/configure_init_service.rs

```diff
@@ -93,8 +93,7 @@ impl ConfigureInitService {
             InitSystem::Systemd => {
                 // If /run/systemd/system exists, we can be reasonably sure the machine is booted
                 // with systemd: https://www.freedesktop.org/software/systemd/man/sd_booted.html
-                if !(Path::new("/run/systemd/system").exists() || which::which("systemctl").is_ok())
-                {
+                if !Path::new("/run/systemd/system").exists() {
```

Comment:
Do we really not want to check the existence of a `systemctl` binary? I thought someone had said that if you switch between WSL1 and 2 that the `/run/systemd/system` directory could exist while not actually running in systemd mode?

Reply:
Let's make it an `&&`